modules/imaging_browser/jsx/ImagePanel.js

```diff
@@ -39,13 +39,13 @@ var ImagePanelHeader = React.createClass({
                         Header Info
                         </button>
                     <span className="caret"></span>
-                </div>
-            </div>
+                </span>
+            </span>
             );
     return (
 
-            <div className="panel-heading">
-            <input type="checkbox" data-file-id={this.props.FileID}
+            <div className="panel-heading clearfix">
```

Comment:
It is necessary. Without it, it breaks on FF and Chrome. I don't know why. I am no front-end guy. I just Googled and it worked.